toolkit/scripts/sodiff/mariner-sodiff.sh

```diff
@@ -2,13 +2,36 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT License.
 
-# Required binaries:
-# rpm and dnf
+sodiff_script_error=false
+while getopts "r:f:v:o:e:" opt; do
+    case $opt in
+        r) rpms_folder="$OPTARG";;
```

Comment:
Since we're here, I'm curious if we couldn't also re-name this to azurelinux-sodiff.sh or something along those lines. Just a P2 suggestion, though.

Reply:
This may be worth doing at a later time, for now I will leave it as is to not break the pipelines at the moment since it will require a split in logic between 2.0 and 3.0 builds creating some unexpected instability.